docs/community/20221115-managed-kubernetes.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Feature Group for Managed Kubernetes in Cluster API
+authors:
+  - "@jackfrancis"
+reviewers:
+    - "@pydctw”"
+    - "@richardcase"
+    - "@vincepri"
+    - "@sbueringer”"
+    - "@fabriziopandini"
+    - "@killianmuldoon"
+creation-date: 2022-11-15
+last-updated: 2022-12-06
+status: proposed
+see-also:
+  - https://github.com/kubernetes-sigs/cluster-api/issues/7494
+  - https://github.com/kubernetes-sigs/cluster-api/issues/6988
+---
+# Managed Kubernetes in Cluster API Feature Group
+
+This document briefly outlines the scope, communication media, and stakeholders for a formal Feature Group dedicated to defining a Cluster API-approved Managed Kubernetes solution.
```

Comment:
>Cluster API-approved Managed Kubernetes solution

@jackfrancis This term in the CAPI context always results confusing to me as it might mean two different things for a reader:
1 - As a service provider I want to use CAPI to power my managed kubernetes offering.
2 - As service consumer I want to use CAPI to request an instance of a Kubernetes Clusters that falls under the managed Kubernetes category, e.g eks.

It'd be good to define the term here to remove ambiguity and clarify this is for 2.

Reply:
@jackfrancis I didn't review the initial version but I can't see a difference. Am I missing something? :)

Reply:
A suggestion:
> for a formal Feature Group dedicated to defining a Cluster API-approved solution to manage clusters with Managed Kubernetes services.

Reply:
Integrated your suggestion, thanks!